solutions/security/ai/ease/ease-alerts.md

```diff
@@ -0,0 +1,36 @@
+---
+navigation_title: Triage alerts
+applies_to:
+  serverless:
+    security: preview
+---
+
+# Triage alerts in EASE
+
+Once you've ingested your alerts to EASE, you can view, track, and analyze them from the **Alert summary** page.
```

Comment:
You might want to spell out EASE since it's the first occurrence on the page. Might be ok since it's spelled out in the nav. Same with first occurrences on the other new pages.